README.md

```diff
@@ -83,3 +83,20 @@ In particular, the
 functions.
 An example notebook can be found at [`pytket-qujax_heisenberg_vqe.ipynb`](https://github.com/CQCL/pytket/blob/main/examples/pytket-qujax_heisenberg_vqe.ipynb).
 
+
+## Contributing
+Bugs and feature requests are managed using [GitHub issues](https://github.com/CQCL/qujax/issues).
+
+Pull requests are welcomed!
+1. First fork the repo and create your branch from [`develop`](https://github.com/CQCL/qujax/tree/develop).
+2. Add your code.
+3. Add your tests.
+4. Update the documentation if required.
+5. Issue a pull request into [`develop`](https://github.com/CQCL/qujax/tree/develop).
+
+New commits on [`develop`](https://github.com/CQCL/qujax/tree/develop) will then be merged into
+[`main`](https://github.com/CQCL/qujax/tree/develop) on the next release.
+
+We use [semantic versioning](https://semver.org/) with version number `x.y.z` -
```

Comment:
Is this what you have planed to do? I would recommend staying with 0.x.y until you have established a stable api?

Reply:
removed comment on versioning